pkgs/by-name/mi/micro/package.nix

```diff
@@ -0,0 +1,23 @@
+{lib}:
+
+micro = buildGoModule rec {
+  pname = "micro";
+  version = "2.0.13";
+
+  src = fetchFromGitHub {
```

Comment:
I'm quite confused because https://nixos.org/manual/nixpkgs/stable/#sec-language-go says to put `hash = "sha256-something";` but it built just fine without it. What am I doing wrong?